flux-accounting-guide.rst

```diff
@@ -148,4 +148,17 @@ association
 bank
   An account that contains associations.
 
+
+.. note::
+
+ Much of the terminology and concepts present in flux-accounting is
+ similar to that of `Slurm`_. The reason for this is to meet and maintain local
+ management requirements, as both were developed at LLNL. The terms and
+ concepts adopted from Slurm are taken with guidance from the `GNU General
+ Public License`_.
+
 .. _documentation: https://sqlite.org/omitted.html
+
+.. _Slurm: https://slurm.schedmd.com/documentation.html
```

Comment:
Rewording suggestion:

> The design of flux-accounting was driven by LLNL site requirements.  Years ago, the design of the Slurm accounting (citation), and its multi-factor priority plugin (citation)  were driven by similar LLNL site requirements.    We chose to reuse terminology and concepts from Slurm to facilitate a smooth transition to Flux.  The flux-accounting code base is all completely new, however.

https://slurm.schedmd.com/accounting.html

https://slurm.schedmd.com/priority_multifactor.html

Reply:
Thanks for this suggestion! I just pushed up a fix that includes your rewording recommendation.